src/main/java/org/opentripplanner/graph_builder/module/osm/OsmModuleBuilder.java

```diff
@@ -24,6 +25,7 @@ public class OsmModuleBuilder {
   private boolean staticParkAndRide = false;
   private boolean staticBikeParkAndRide = false;
   private int maxAreaNodes;
+  private StreetLimitationParameters streetLimitationParameters;
```

Comment:
I would vote to set a default value here and check in `OsmModule`'s constructor that it definitely isn't null.